ibm/resource_ibm_is_vpc_address_prefix.go

```diff
@@ -303,14 +317,17 @@ func resourceIBMISVpcAddressPrefixUpdate(d *schema.ResourceData, meta interface{
 		name = d.Get(isVPCAddressPrefixPrefixName).(string)
 		hasChanged = true
 	}
-
+	if d.HasChange(isVPCAddressPrefixDefault) {
```

Comment:
I think the update will override the value of is_default...Let take scenario user created intially with is_default true...after successfull apply he just update the name....
isDefault := false this value will be passed to update even there is no change

Reply:
added a separate check for isDefault and name